Source/WebCore/css/parser/CSSParserImpl.cpp

```diff
@@ -651,6 +651,11 @@ RefPtr<StyleRuleKeyframes> CSSParserImpl::consumeKeyframesRule(CSSParserTokenRan
     AtomString name;
     if (nameToken.type() == IdentToken) {
         name = nameToken.value().toAtomString();
+        // From the CSS Values specification, keyframe names that are identifiers
+        // are not allowed to be CSS wide keywords. Nor can they be "default". And CSS Animations
+        // additionally excludes the "none" keyword.
+        if (isValidCustomIdentifier(nameToken.id()) || nameToken.id() == CSSValueNone)
```

Comment:
```suggestion
        if (!isValidCustomIdentifier(nameToken.id()) || nameToken.id() == CSSValueNone)
```

The tests are failing because of this.

Reply:
(sorry looks like my original suggestion had the typo)